packages/docs-theme/src/components/example.tsx

```diff
@@ -114,6 +121,9 @@ export class Example extends React.PureComponent<IDocsExampleProps> {
     }
 
     public componentDidMount() {
+        if (this.props.forceUpdate === false) {
```

Comment:
this is very unintuitive. What about using default props or destructuring with a default and `if (!this.props.forceUpdate)`?